pkg/reconciler/workload/namespace/namespace_reconcile_scheduling.go

```diff
@@ -37,30 +33,24 @@ import (
 
 const removingGracePeriod = 5 * time.Second
 
-// placementSchedulingReconciler reconciles the state.workload.kcp.io/<syncTarget> labels according the
+// reconcileScheduling reconciles the state.workload.kcp.io/<syncTarget> labels according the
```

Comment:
super-nit: this comment (and a bunch of the others) are mechanistic, but would be more useful if they were semantic - what does it _mean_ to move an annotation from a placement to a label on a synctarget? What's the point of what we're doing here?

Reply:
"I'm just refactoring existing code" 😛 but let me see if I can clarify

Reply:
I know you are - hence, super-nit :)

Reply:
Filed as #2896